### ISSUE TITLE
-sceneList is ignored in UWP

### ISSUE THREAD

**Issue by djee-ms** [microsoft/MixedRealityToolkit-Unity#9677]
`UnityPlayerBuildTools.cs` is parsing the argument:
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/265a0ea3621c84b88e4b02e7fcd1b5682afef56a/Assets/MRTK/Core/Utilities/BuildAndDeploy/UnityPlayerBuildTools.cs#L226-L228

But `UwpPlayerBuildTools.cs` is not:
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/265a0ea3621c84b88e4b02e7fcd1b5682afef56a/Assets/MRTK/Core/Utilities/BuildAndDeploy/UwpPlayerBuildTools.cs#L29-L44

Instead, on UWP, it's relying on the project settings:
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/265a0ea3621c84b88e4b02e7fcd1b5682afef56a/Assets/MRTK/Core/Utilities/BuildAndDeploy/UwpPlayerBuildTools.cs#L63

This means the option is not cross-platform, and we are stuck in CI without a solution to build our AppX with a specific scene in UWP.

**Comment by MaxWang-MS** [comment]
Hi @djee-ms, I just took a look at the relevant files but think we are using the scenes passed in by the -sceneList flag in UWP builds.

In `UwpPlayerBuildTools` we are reading the command line arguments as follows:
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/265a0ea3621c84b88e4b02e7fcd1b5682afef56a/Assets/MRTK/Core/Utilities/BuildAndDeploy/UwpPlayerBuildTools.cs#L112-L119

And in the `ParseBuildCommandLine` function we call into `UnityPlayerBuildTools.ParseBuildCommandLine` which should read in the scenes in the command line argument properly. Please let me know if that is not the case.

**Comment by djee-ms** [comment]
Well it looks like you're right, I hadn't seen the call to `UnityPlayerBuildTools.ParseBuildCommandLine`. But when I tried I ended up with the scene which is configured inside the editor config, and not the one I passed on the command line with `-sceneList`. Is that because the scene list does a union instead of overwriting the selected scenes? Because in any case if there's a `-sceneList` flag I expect this is the list of scenes built, not a subset of it.
